kafka-connect-fitbit-source/src/main/java/org/radarbase/connect/rest/fitbit/FitbitRestSourceConnectorConfig.java

```diff
@@ -849,7 +878,7 @@ public Duration getPollIntervalPerUser() {
   }
 
   public Duration getTooManyRequestsCooldownInterval() {
-    return Duration.ofHours(1);
+    return Duration.ofSeconds(getInt(FITBIT_TOO_MANY_REQUESTS_COOLDOWN_CONFIG));
```

Comment:
Here we have an issue of problems with backwards compatibility I think. We should take care to add some migration note to RADAR-Kubernetes.

Reply:
What do you mean? The default is set to 1hr still so should not change any functionality in existing deployments.